libs/database-entities/src/main/java/gov/cdc/nbs/entity/syphilis_qb/QuestionGroup.java

```diff
@@ -0,0 +1,36 @@
+package gov.cdc.nbs.entity.syphilis_qb;
+
+
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.OneToMany;
+import javax.persistence.FetchType;
+
+
+@Entity
+@Table(name = "question_group")
+public class QuestionGroup {
+    @Id
+    @GeneratedValue
+    @Column(name = "question_set_uid", nullable = false)
+    private Long questionGroupId;
+
+    @Column(name = "questions_question_uid", nullable = false)
+    private Long questionId;
+
+    @Column(name = "condition_set_condition_uid")
+    private Long conditionId;
+
+    @Column(name = "questionnaire_questionnaire_id")
+    private Long questionnaireId;
```

Comment:
Lets remove the tie from `QuestionGroup` to `Questionnaire`. Similar to how questions aren't tied to a specific group, groups don't need to be tied to specific `Questionnaires`. This will allow us to reuse `QuestionGroups` in multiple `Questionnaires`